dvc/cache.py

```diff
@@ -42,51 +42,50 @@ def __init__(self, repo):
                 Config.SECTION_CACHE_PROTECTED: protected,
             }
 
-        self._local = Remote(repo, sect)
+        self.local = Remote(repo, settings)
+        self.s3 = self._get_remote(config, Config.SECTION_CACHE_S3)
```

Comment:
I removed the `properties` and replace them with non-underscore values because I think it doesn't bring any value.

Reply:
@mroutis Please do such refactorings in a separate commit next time, otherwise, it obstructs the view.